### PR TITLE
feat(tests): web3 wrapper test

### PR DESCRIPTION
**Pull request by TheNaturator**

### CODE REVIEW

File: tests/integration/kleros.test.js [hunk 1]
```diff
@@ -31,4 +35,35 @@ describe('Kleros', () => {
...
+        .toBigNumber('200000000000000000000001')
+        .toString(10)
+    ).toBe('200000000000000000000001')
+    // Balance/user functions
+    expect(klerosInstance.web3Wrapper.getBalanceInWeiAsync()).toBeTruthy()
+    expect(
+      klerosInstance.web3Wrapper.doesContractExistAtAddressAsync(
+        randomEthAddress
+      )
+    ).resolves.toBeTruthy()
```
Comment (satello) [inline tests/integration/kleros.test.js]: How does this test case pass? The random ETH address shouldn't be a contract on the test blockchain

File: tests/integration/kleros.test.js [hunk 2]
```diff
@@ -31,4 +35,35 @@ describe('Kleros', () => {
...
+    // Block tests
+    expect(klerosInstance.web3Wrapper.getBlock(10)).resolves.toBeTruthy()
```
Comment (satello) [inline tests/integration/kleros.test.js]: This one will be a race condition depending on what order the tests execute it. There might not be a block at height 10 if this test gets executed first

**Review by satello — COMMENTED** [review]
Thanks for writing this test. I am a little confused how this passes. Perhaps they need some stricter checks than `toBeTruthy()` as several of them seem like they should be failing

File: tests/integration/kleros.test.js (hunk 1 above)
Comment (TheNaturator) [reply tests/integration/kleros.test.js]: I agree, that is just silly.

File: tests/integration/kleros.test.js (hunk 2 above)
Comment (TheNaturator) [reply tests/integration/kleros.test.js]: I also agree in case of ganache. On normal blockchain should be the 10 block but I will change it  anyway.

**Pull request closed without merging**